src/docs/sdk/event-payloads/breadcrumbs.mdx

```diff
@@ -2,107 +2,146 @@
-  [description of recognized breadcrumb types](#breadcrumb-types).
+`category` (optional)
 
-`category`
+A dotted string indicating what the crumb is or from where it comes. Typically it is a module name or a descriptive string. For instance, `ui.click` could be used to indicate that a click happened in the UI or flask could be used to indicate that the event originated in the Flask framework.
 
-: _Optional_. A dotted string indicating what the crumb is or from where it
-  comes. Typically it is a module name or a descriptive string. For instance,
-  _ui.click_ could be used to indicate that a click happened in the UI or
-  _flask_ could be used to indicate that the event originated in the Flask
-  framework.
+Internally we render some crumb’s color and icon based on the provided category. For more information, see the [description of recognized breadcrumb types](#breadcrumb-types).
 
-`message`
+`message` (optional)
 
-: _Optional_. If a message is provided, it is rendered as text with all
-  whitespace preserved. Very long text might be truncated in the UI.
+Human readable message for the breadcrumb.
 
-`data`
+If a message is provided, it is rendered as text with all whitespace preserved.
 
-: _Optional_. Arbitrary data associated with this breadcrumb. Contains a
-  dictionary whose contents depend on the breadcrumb `type`. See the
-  [description of breadcrumb types](#breadcrumb-types). Additional parameters
-  that are unsupported by the type are rendered as a key/value table.
+`data` (optional)
 
-`level`
+Arbitrary data associated with this breadcrumb.
 
-: _Optional_. This defines the severity level of the breadcrumb. Allowed values
-  are, from highest to lowest: `fatal`, `error`, `warning`, `info`, and `debug`.
-  Levels are used in the UI to emphasize and deemphasize the crumb. Defaults to
-  `info`.
+Contains a dictionary whose contents depend on the breadcrumb type. Additional parameters that are unsupported by the type are rendered as a key/value table.
+
+`level` (optional)
+
+This defines the severity level of the breadcrumb. Allowed values are, from highest to lowest: `fatal`, `error`, `warning`, `info`, and `debug`. Levels are used in the UI to emphasize and deemphasize the crumb. Defaults to `info`.
+
+`timestamp` (recommended)
+
+A timestamp representing when the breadcrumb occurred. The format is either a string as defined in [RFC 3339](https://tools.ietf.org/html/rfc3339) or a numeric (integer or float) value representing the number of seconds that have elapsed since the [Unixepoch](https://en.wikipedia.org/wiki/Unix_time).  
+Breadcrumbs are most useful when they include a timestamp, as it creates a timeline leading up to an event `expection/error`.
 
 ## Breadcrumb Types
 
-Below are descriptions of individual breadcrumb types, and what their `data`
-properties look like.
+Below are descriptions of individual breadcrumb types, and what their `data` properties look like.
 
 ### `default`
```

Comment:
rather than setting these as H3, I'd recommend using 
`term`
: description

as the format for this and the following terms for easier readability. You'll need to move the margin in for the code samples so they align as well.

Reply:
I tried to align the samples and phrases that start with "Internally ..." by adding some indentation, but that doesn't seem possible as it still displays the same or the json as a text. Please see below:

![image](https://user-images.githubusercontent.com/29228205/91662043-299dc200-eae0-11ea-80bc-e28d008ce59b.png)

Reply:
Hi @priscilawebdev, thanks for helping with docs. I'm taking a look at this and may have a solution shortly. 😸